moveit_msgs/CollisionObject bin_to_place
---
bool empty
